pages/common/az-login.md

```diff
@@ -0,0 +1,29 @@
+# az login
+
+> Login to Azure.
```

Comment:
```suggestion
> Log in to Azure.
```
When used as a verb it's `log in` not `login`. This should be changed in all descriptions